api/tests/test_hyp3_api.py

```diff
@@ -71,28 +73,90 @@ def test_submit_job(client, states_stub):
     }
 
 
+@mock_dynamodb2
 def test_list_jobs(client):
+    table_name = environ['TABLE_NAME']
```

Comment:
don't need a local variable here, can just to `TableName=environ['TABLE_NAME']` later